packages/client/test/integration/pow.spec.ts

```diff
@@ -18,7 +23,7 @@ async function setupPowDevnet(prefundAddress: Address, cleanStart: boolean) {
   const addr = prefundAddress.toString().slice(2)
   const consensusConfig = { ethash: true }
 
-  const defaultChainData = {
+  const defaultChainData: Omit<GethGenesis, 'alloc'> = {
```

Comment:
Same nit here: should we default `alloc = {}` here instead of omitting it from the type?